.github/workflows/ci.yml

```diff
@@ -77,6 +78,20 @@ jobs:
         # bench.yml workflow runs benchmarks only on linux.
         if: startsWith(matrix.os, 'ubuntu')
 
+  test-parking_lot:
+    name: compile tests with parking lot send guards
```

Comment:
might be nice to have a brief comment above this step explaining what _exactly_ this is doing and why? not a blocker.